File: Easy/26-remove-duplicates-from-sorted-array/try-8.cpp
/*
 * Problem #26: Remove Duplicates from Sorted Array
 * Difficulty: Easy
 * Submission: Try 8
 * status: Accepted
 * Language: cpp
 * Date: 30/11/2024, 12:05:10
 * Link: https://leetcode.com/problems/remove-duplicates-from-sorted-array/
 */

class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int ind=1,i;
        for(i=0;i<nums.size()-1;i++){
            if(nums[i]!=nums[i+1]){
                nums[ind++]=nums[i+1];
            }
        }
        return ind;
    }
};

File: Easy/242-valid-anagram/try-8.py
/*
 * Problem #242: Valid Anagram
 * Difficulty: Easy
 * Submission: Try 8
 * status: Accepted
 * Language: python3
 * Date: 06/12/2024, 15:47:36
 * Link: https://leetcode.com/problems/valid-anagram/
 */

class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        return sorted(s)==sorted(t)
        
